### PR TITLE
More vaults!

### PR DESCRIPTION
**Pull request by PJB3005**
**LITERALLY 0% TESTED I'LL DO THAT SOON**

### CODE REVIEW

**Comment by Imagebard** [comment]
What the hell is so important on the 2nd vault that they had to put 6 syndies in it?

**Comment by SkowronX** [comment]
Guns and ammo?

**Comment by PJB3005** [comment]
Also it should be noted that anything a syndie drops is also loot, technically. 

But yeah I'm open to suggestions on the loot for the listening outpost.

**Comment by icantthinkofanameritenow** [comment]
The second one is a bit too similar to the Ruskie DJ station, if you ask me.  Giving it its own SMES and APC doesn't seem like the best idea either, since that could mean more unfixable alarms to annoy the sillycones.

**Comment by PJB3005** [comment]
@Icantthinkofanameritenow I made the APC not output power alarms.

**Comment by icantthinkofanameritenow** [comment]
@PJB3005
Yeah, I just noticed that in the code.

Still, the layout could use with a but of modification, since right now it still looks a bit too much like the DJ station.

**Comment by Shadowmech88** [comment]
Why is it a bad thing for it to look similar to the DJ station?

**Comment by PJB3005** [comment]
It's intended to be a more serious syndicate listening outpost, I could add an armoury to it along with some unbreakable walls to make it stand out and make it more rewarding to kill the syndicates.

**Comment by ihadtoregisterforthis** [comment]
And I suppose the radio machinery doesn't broadcast for everyone nearby, right?

**Comment by icantthinkofanameritenow** [comment]
@Shadowmech88 
I can't explain it too well, it just feels...overly familiar, as it were. Like I've been there before already plenty of times. 

@PJB3005 
I think my own Syndicate-themed vault more than covers that niche. Perhaps you could try using the pirate mobs to replace the Syndicate ones, just to mix things up- they seem similar enough and their loot should be just about identical.

**Comment by Imagebard** [comment]
But pirates have no reason to listen to Nanotrasen.
Either way, it just looks like a syndicate mini-base, which is what icantthinkofanameritenow did. Make the telecomms room closer to the syndie next to the radios, put some spare machinery there.
As for loot, I really have no idea.

**Comment by PJB3005** [comment]
@Icantthinkofanameritenow yours is huge though, mine is more of a small outpost with some basic defenses. 

@ihadtoregisterforthis no the relay starts off by default.

**Comment by icantthinkofanameritenow** [comment]
@Imagebard 
Well, mine is a base but there's very little about it that can be called "mini".

**Comment by Imagebard** [comment]
@icantthinkofanameritenow What is the point of having a "syndicate base" and a "mini syndicate base" in the current rotation? Maybe when there is a bigger variety of vaults.

**Comment by icantthinkofanameritenow** [comment]
@Imagebard 
Can you rephrase that? I'm not sure what you mean by that.

**Comment by Imagebard** [comment]
Eh, doesn't matter.

**Comment by Intigracy** [comment]
tbh you should have a syndicate telecomms thing and a few headsets like nuke ops as loot.

**Comment by icantthinkofanameritenow** [comment]
@Intigracy
I second that. The Syndicate communications keys spawn all the time on Defficiency, so we know it won't exactly be gamebreaking.

**Comment by gbasood** [comment]
> The Syndicate communications keys spawn all the time on Defficiency, so we know it won't exactly be gamebreaking.

Honestly I feel like that should absolutely not happen but it's unrelated to this

**Comment by PJB3005** [comment]
Wow I just realised I had this PR open, I should fix it.

**Comment by clusterfack** [comment]
Yeah, you could

**Comment by PJB3005** [comment]
Fixed

**Comment by PJB3005** [comment]
Syndicate numbers on the listening outpost has been reduced quite a lot, added 3 all frequencies (except syndicate and binary) centcomm encryption keys.

**Comment by clusterfack** [comment]
I hope you didn't think your PR would escape from conflicts unscathed

**Comment by PJB3005** [comment]
I didn't it just caught the conflict flu again.

**Comment by ghost** [comment]
If the worry is that the syndicate mobs would mean there would be plenty of loot, why not add to the custom items for vault dm a child of the syndicate mobs where, on death, they explode a-la nuclear operative explosive implant.

**Comment by ComicIronic** [comment]
Did you take this from 0% tested to 100% tested?

**Comment by PJB3005** [comment]
Yes.

**Comment by ComicIronic** [comment]
Why not
